recipe-server/client/control/tests/components/data/test_QueryExperiments.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import TestComponent from 'control/components/data/QueryExperiments';
+
+const { WrappedComponent: QueryExperiments } = TestComponent;
+
+
+describe('<QueryExperiments>', () => {
+  const props = {
+    fetchExperiments: () => {},
+  };
+
+  it('should work', () => {
+    const wrapper = () => shallow(<QueryExperiments {...props} />);
+
+    expect(wrapper).not.toThrow();
+  });
+  it('should call fetchExperiments on componentWillReceiveProps', () => {
+    let called = false;
+    const experimenterAPIUrl = 'https://localhost/api/v1';
+    const fetchExperiments = calledExperimenterAPIUrl => {
+      called = calledExperimenterAPIUrl === experimenterAPIUrl;
```

Comment:
You could probably use a stub for this, instead of rolling your own.

```js
const fetchExperimentsSpy = jasmine.createSpy();
/// mount
expect(fetchExperimentsSpy).not.toHaveBeenCalledWith(experimenterAPIUrl);
/// set props
expect(fetchExperimentsSpy).toHaveBeenCalledWith(experimenterAPIUrl);
```